src/Components/Bookpage/BookDetails/Reviews/index.js

```diff
@@ -0,0 +1,15 @@
+import "./reviews.css"
+
+function Reviews({ review }) {
+    return (
+        <>
+            <div>
+                <p className="reviewer">{review.name}</p>
```

Comment:
Semantically better for this to be an (h4) and you can also get rid of the class